src/sgame/sg_missile.cpp

```diff
@@ -517,7 +528,9 @@ static trace2_t MissileTrace( gentity_t *ent )
 	return result;
 }
 
-void G_RunMissile( gentity_t *ent )
+// Move missile along its trajectory and detect collisions.
+// Returns true if the missile has ceased to exist
+bool G_MoveMissile( gentity_t *ent )
```

Comment:
I would have expected this function and all others to live in the missile component?

Reply:
The current working version of my [missile CBSE branch](https://github.com/slipher/Unvanquished/tree/mis-cbse) has that code moved into the missile component. Just trying to break it up into smaller pieces